configs/fine-tune-llama2/aws.yaml

```diff
@@ -0,0 +1,10 @@
+head_node_type:
+  name: head_node_type
+  instance_type: m5.xlarge
+
+worker_node_types:
+- name: gpu_worker
+  instance_type: g5.4xlarge
+  min_workers: 0
+  max_workers: 100
```

Comment:
This is an arbitrary number. I just give a high enough number so that users won't run into autoscaling limit issue if they custom stuff or run something not related to this template...